.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "solr-js"]
```

Comment:
Next iteration i'd like to get rid of the submodule and just add the logic of pulling down the solr-js repo as part of the build.  We could put all the logic for "building for prod" in a makefile even